python/ray/data/datasource/datasource.py

```diff
@@ -18,20 +18,21 @@
 
 @PublicAPI
 class Datasource:
-    """Interface for defining a custom ``ray.data.Dataset`` datasource.
+    """Interface for defining a custom :class:`~ray.data.Dataset` datasource.
 
-    To read a datasource into a dataset, use ``ray.data.read_datasource()``.
-    To write to a writable datasource, use ``Dataset.write_datasource()``.
+    To read a datasource into a dataset, use :meth:`~ray.data.read_datasource`.
+    To write to a writable datasource, use :meth:`~ray.data.Dataset.write_datasource`.
 
-    See ``RangeDatasource`` and ``DummyOutputDatasource`` for examples
+    See :class:`~ray.data.datasource.RangeDatasource` and
+    :class:`~ray.data.datasource.DummyOutputDatasource` for examples
     of how to implement readable and writable datasources.
```

Comment:
These aren't included in our API reference, so these links are dead.
```suggestion
```

Reply:
should we still include the comment as is then? so that readers of the code know to reference these classes